src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Azure Site Recovery added support for protecting virtual machines with target virtual machine scale set configured for Azure to Azure provider.
```

Comment:
nit: Azure Site Recovery support for Virtual Machine Scale Set for Azure to Azure provider.

Reply:
done